src/libs/ajax/workspaces/Workspaces.ts

```diff
@@ -202,7 +203,7 @@ export const Workspaces = (signal?: AbortSignal) => ({
         return res.json();
       },
 
-      importMethodConfig: (config) => {
+      importMethodConfig: (config: MethodConfiguration) => {
```

Comment:
Please also add the return type.

Reply:
Added!

Reply:
Added!

Reply:
You are parsing this response and therefore know the structure of the data it will include. So instead of `Response`, can you please define an interface that describes the expected data format? If you look at other methods in `Worksapces.ts` with return types, it may help to understand what I'm asking.

As a best practice, I've been trying to name these interfaces consistently with what you see in Rawls Swagger. So the return interface for this method would be `ValidatedMethodConfiguration`: https://rawls.dsde-dev.broadinstitute.org/#/methodconfigs/create_method_configuration.

Reply:
Or perhaps I'm misreading the method `makeExportWorkflowFromMethodsRepoProvider`. It is declared as returning a `ExportWorkflowToWorkspaceProvider`, and within that code there is a return from the `importMethodConfig` Ajax method, so I thought that the returned JSON from `importMethodConfig` was being used. Perhaps that is not true, in which case a return type of `Response` is fine (since you aren't depending on the structure of the return type).

Reply:
Neither the preexisting usage of `importMethodConfig` nor my new usage relies on the return type of the API response. The `export` method of the provider created by `makeExportWorkflowFromMethodsRepoProvider` is ultimately called [here](https://github.com/DataBiosphere/terra-ui/blob/6efc3a4833b9f487ca70e2f591b71f21149ff763/src/workflows/modals/ExportWorkflowModal.tsx#L126), and the response of the Ajax call is not used.